main.cpp

```diff
@@ -381,11 +472,22 @@ int main(int argc, char * argv[]) {
 
 	SkyblivionConverter converter = SkyblivionConverter(oblivionCollection, skyrimCollection, std::string(argv[3]));
 
-	std::cout << "Converting QUST records.. " << std::endl;
-	std::vector<Sk::QUSTRecord *> *resQUST = converter.convertQUSTFromOblivion();
 	std::cout << "Converting DIAL records.. " << std::endl;
 	std::vector<Sk::DIALRecord *> *resDIAL = converter.convertDIALFromOblivion();
 
+	/**
```

Comment:
Why changed order? :)

Reply:
If some of the topics are added after quests, when converting quests they do not exist in EDID map. So they are not being bound. While this is not related to adding not referenced actors, to solve the issue with ArenaSpectator quest the topic related to it needs to be bound.

Reply:
Makes sense - blank QUST records are here in .esm, so converting DIALs first will bind them properly.
Thanks